controller.py

```diff
@@ -0,0 +1,43 @@
+from typing import List, Dict
+
+from db.iclient_db import IClientDB
+from entities.card import Card
+from entities.card_proprties import CardProperties
+from entities.status import Status
+
+from flask import Flask, request
+from consts import *
+
+
+CARDS_SERIALIZED = Dict[str, List[Dict[str, str]]]
+
+
+class Controller:
+    def __init__(self, client_db: IClientDB):
+        self.app = Flask(__name__)
+        self.client_db = client_db
+        self.setup_routes()
+
+    def setup_routes(self):
+        self.app.route("/add_card", methods=[POST])(self.add_card)
+        self.app.route("/remove_card", methods=[POST])(self.remove_card)
+        self.app.route("/filter_cards", methods=[POST])(self.filter_cards)
+
+    def run(self):
+        self.app.run(debug=True)
+
+    def add_card(self) -> Status:
+        card = Card.generate_card(request)
```

Comment:
What is request? 
Why doesn't a function parameter? 
How do I validate this function is called only on http request. 
Consider renaming it to make it more explict.

Reply:
This comment is valid for all the http methods in this file